lib/converters/v2.1.0/converter-v21-to-v1.js

```diff
@@ -0,0 +1,128 @@
+var _ = require('lodash').noConflict(),
+    util = require('../../util'),
+    inherits = require('inherits'),
+
+    v2Common = require('../common/v2'),
+    BaseBuilders = require('../v2.0.0/converter-v2-to-v1').Builders,
+
+    Builders;
+
+inherits(Builders = function ConversionBuilder () {
+    Builders.super_.apply(this, arguments);
+}, BaseBuilders);
+
+_.extend(Builders.prototype, {
+    /**
+     * Converts a V2.1 request to a V1 request.
+     *
+     * @param {Item|Object} item - The wrapped request instance to be converted.
+     * @param {String} collectionId - The collection id to be added to the converted v1.0.0 compliant request.
+     * @param {Boolean} skipResponses - Will result in saved responses being ignored for conversion if set to true.
+     * @returns {Object} An equivalent v1.0.0 representation for the current v2.0.0.1 request.
+     */
+    request: function (item, collectionId, skipResponses) {
+        var _item = _.cloneDeep(item), // do not mutate the original
+            auth = _.get(_item, 'request.auth');
+
+        if (auth && auth.type && _.isArray(auth[auth.type])) {
+            auth[auth.type] = _.transform(auth[auth.type], function (a, element) {
+                a[element.key] = element.value;
```

Comment:
we don't do type conversion or force string here?

Reply:
No, some parameters can be boolean as well. (Those are primarily UI based)

Reply:
question is when we do v1 to v2, do we use the original type to set type?